demos/graph-classification/supervised-graph-classification.ipynb

```diff
@@ -93,7 +93,8 @@
     "from tensorflow.keras.layers import Dense\n",
```

Comment:
<p>Does it make sense to collect all the histories into a list and do an "ensemble" history plot? I think <code>sg.utils.plot_history</code>  can take a list of histories instead of a single one and plot them all together.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/stellargraph/stellargraph/pull/1387/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
I'll have a look but the plot will look busy and noisy as there is much variance in performance based on the fold.

Reply:
Yeah, as I thought plotting all the histories looks ugly and unhelpful. We best leave this bit for the DGCNN notebook that demonstrates performance on a single split of the data.

![image](https://user-images.githubusercontent.com/33439627/80443100-ca714a80-8951-11ea-94f5-c82a927c2e23.png)